newIDE/app/src/GameDashboard/GameDetailsDialog.js

```diff
@@ -167,6 +173,29 @@ export const GameDetailsDialog = ({
     }
   };
 
+  const unpublishGame = React.useCallback(
+    async () => {
+      if (!profile) return;
+
+      const { id } = profile;
+      try {
+        setPublicGame(null); // Public game will auto update when game is updated.
```

Comment:
This comment can lead to confusion. I wondered it it means that a public game is always updated to Liluo.
It's public game fields that are updated in the code, right?

Reply:
yes good point, I'll make it clearer 👍